examples/ex22.cpp

```diff
@@ -282,6 +290,11 @@ int main(int argc, char *argv[])
    ConstantCoefficient negMassCoef(omega_ * omega_ * epsilon_);
 
    SesquilinearForm *a = new SesquilinearForm(fespace, conv);
+   if (pa)
+   {
+      a->real().SetAssemblyLevel(AssemblyLevel::PARTIAL);
+      a->imag().SetAssemblyLevel(AssemblyLevel::PARTIAL);
```

Comment:
I'd prefer it if we create a `SetAssemblyLevel` member function in `[Par]SesquilinearForm`.  If you think it would be useful, we could provide two functions: one assuming the same level for the real and imaginary parts and one with separate levels.

Reply:
Yes, that's a good idea. I'm not sure about the use case for having different assembly levels for real and imaginary parts of the same sesquilinear form. I can add it if there is a need for that.

Reply:
No, I don't think we need separate assembly types.  I just wanted to mention it as a possibility.  If someone really needs that they can always set them as you do above.

Reply:
I have now added a `[Par]SesquilinearForm::SetAssemblyLevel` function that sets the assembly level equally for both its real and imaginary `[Par]BilinearForm`.